src/main/scala-2.13/org/apache/james/gatling/jmap/rfc8621/scenari/PushPlatformValidationScenario.scala

```diff
@@ -18,16 +18,18 @@ object PushPlatformValidationScenario {
   val emailState = "emailState"
   val inbox = "inboxID"
   val draft = "draftMailboxId"
+  val spam = "spamMailboxId"
   val outbox = "outboxMailboxId"
   val accountId = "accountId"
   val emailIds = "emailIds"
+  val emailId = "emailId"
   val messageIds = "messageIds"
   val randomMailbox = "randomMailbox"
 }
 
 class PushPlatformValidationScenario(minMessagesInMailbox: Int,
-                                     minWaitDelay: FiniteDuration = 20 seconds,
-                                     maxWaitDelay: FiniteDuration = 40 seconds) {
+                                     minWaitDelay: FiniteDuration = 2 seconds,
+                                     maxWaitDelay: FiniteDuration = 4 seconds) {
```

Comment:
Q: We should revert the pause duration after the perf test MOVE, no? Otherwise, we would put much more stress on our JMAP simulation IMO.

Reply:
Agreed, actually that change should not be in the code base :)